homeassistant/components/netatmo/config_flow.py

```diff
@@ -33,6 +33,7 @@ def extra_authorize_data(self) -> dict:
             "read_station",
             "read_thermostat",
             "write_camera",
+            "write_presence",
```

Comment:
Does this mean that people need to re-authorize? How do we deal with that?

Reply:
Yes, they do. The idea is to schedule a reauthorization when a incomplete scope is detected.